README.md

```diff
@@ -45,9 +46,19 @@ const config = {
   experienceVersion: 'PRODUCTION',
 }
 
+const searcher = provideHeadless(config);
+
+searcher.setSessionTrackingEnabled(true);
+let sessionId = window.sessionStorage.getItem('sessionId');
+if (!sessionId) {
+  sessionId = uuidv4();
+  window.sessionStorage.setItem('sessionId', sessionId);
+}
+searcher.setSessionId(sessionId);
+
 function App() {
   return (
-    <SearchHeadlessProvider {...config}>
+    <SearchHeadlessProvider searcher={searcher}>
```

Comment:
Kind of a tangent but are we planning to get the SearchProvider (combined provider SearchHeadlessProvider and AnalyticsProvider) that Davish talked about in v2? It came to mind since we'd have to update the readme again when adding it

Reply:
I don't really think we have time right now. It needn't be in a V2. We could still keep both the `SearchHeadlessProvider` and `AnalyticsProvder`. We'd be adding a convenience wrapper over them.